content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+    - uri:
+        prefix: /svc-2
+    route:
+    - destination:
+        host: svc-2
+{{< /text >}}
+
+The following diagram shows the configured traffic routes implementing a
+virtual service to configure traffic for services in a namespace for two
+distinct application services based on the URI prefixes:
+
+{{< image width="50%"
+    link="./net-config-5.svg"
+    caption="Configurable traffic rules implementing a namespace for two  application services"
+    >}}
+
+Once you have identified and configured the virtual services suitable for your
+mesh, you can add gateways to route traffic in or out of your mesh. The virtual
+services together with destination rules allow you to configure the behavior of
+the traffic to fulfill the needs of your mesh. Together with a service entry,
+you can use virtual services to configure traffic routes to external
+dependencies.
+
+Visit our [virtual services reference documentation](/docs/reference/config/istio.networking.v1alpha3/#VirtualService) to review all the enabled keys and values.
+
+## Gateways
+
+The Istio gateways are layer 7 load balancers. With an Istio gateway, you can
+configure routes for HTTP and TCP traffic regardless of where the gateway runs.
+Your mesh can have any number of gateways and multiple different gateway
+implementations can co-exist within your mesh. Using a gateway, you can
+configure workload labels to reuse your existing network appliances such as:
+firewall functions, caching, authentication, network address translation, and
+IP address management.
+
+By separating the layer 4 and 6 specifications from the layer 7, the Istio
+gateways overcome the shortcomings of the layer 4 and 6 specifications. Istio
+only configures the functions that layer 7 load balancing proxies implement
+uniformly. To use standard Istio rules to control HTTP requests as well as TCP
+traffic entering a gateway, you must bind a virtual service to the gateway. The
+following example shows a possible gateway configuration to ingress external
+HTTPS traffic to the mesh:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: ext-host-gwy
+spec:
+  servers:
+  - port:
+      number: 443
+      name: https
+      protocol: HTTPS
+    hosts:
+    - ext-host
+    tls:
+      mode: SIMPLE
+      serverCertificate: /tmp/tls.crt
+      privateKey: /tmp/tls.key
+{{< /text >}}
+
+To configure the external HTTPS traffic to flow from the `ext-host` host, you
+must add the Istio gateway to a Istio virtual service as shown:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: virtual-svc
+spec:
+  hosts:
+    - ext-svc
+  gateways:
+    - ext-host-gwy
+{{< /text >}}
+
+The following diagram shows the configured routes for the external HTTPS
+traffic coming from the external host on port 443 to the `virtual-svc` virtual
+service through the `ext-host-gwy` gateway to reach the `my-svc` service
+within the mesh:
```

Comment:
I think that this following diagram shows what the previous 2 YAMLs describe. Again: probably better to start with the prose description of what we want (e.g., "traffic from outside my mesh addressed to https:my-svc should get routed to the service (workload?) called my-svc").

And it would be much better to have something like "shopping-cart" than "my-svc"

Reply:
That is exactly what it shows, I assume the text doesn't make that clear?
The diagrams are an effort to "show" rather than "tell" using prose. I've added a brief sentence though. 
The service can be called whatever but using "my-svc" and similar names makes it clear that these are placeholder names.